GOVERNANCE.md

```diff
@@ -0,0 +1,23 @@
+# Reference Types Working Group Governance
+
+## Discussion
+
+Discussions take place on GitHub's [discussion board](https://github.com/opencontainers/wg-reference-types/discussions) and during [weekly meetings](https://github.com/opencontainers/wg-reference-types/blob/main/README.md#meetings). Notes are taken down at the meetings are recorded in [this google document](https://docs.google.com/document/d/1SVOWQTowigXzbYdorzfa7tMmrcm91yK12LvSONqziJY/edit). Initial seeding of the GitHub discussion topics take place in [this google document](https://docs.google.com/document/d/1ky0LMAvWL_XRnfId9h5GBswV7GGko5GejuyaMWCGX-Y/edit#). It is expected that these google documents will be deprecated in favor of the GitHub Discussion board and GitHub issues.
+
+Discussions shall follow the [documented personas](https://github.com/opencontainers/wg-reference-types/blob/main/docs/PERSONAS.md) previously decided by this working group. Changes to the framework are encouraged via GitHub discussions, issues, and pull requests.
+
+Working group members and maintainers shall follow the [OCI code of conduct](https://github.com/opencontainers/.github/blob/master/CODE_OF_CONDUCT.md)
+
+## Documentation
+
+Meeting minutes shall be published on this GitHub repository. A notetaker shall be identified to take notes during the meetings. The notetaker shall either transfer the notes to the GitHub repository or work with a maintainer to transfer notes. Consensus shall be documented in the form of GitHub issues that can be referenced in pull requests.
```

Comment:
To do this, I'd recommend moving our note taking off of Google Docs into hackmd.io to make PR's easier (copy and paste into a markdown file in the git repo).